pkgs/development/node-packages/node-env.nix

```diff
@@ -56,7 +56,7 @@ let
       ) dependencies);
 
   # Recursively composes the dependencies of a package
-  composePackage = { name, packageName, src, dependencies ? [], ... }:
+  composePackage = { name, packageName, src, dependencies ? [], ... }@args:
```

Comment:
Why this?

Reply:
I only applied the documented way of adding a package. I have no idea why it's so non-compositional and I have no idea why this reappeared.

Reply:
Oh I see, it's auto-generated. Never mind then